pkg/asset/cluster/tfvars.go

```diff
@@ -258,3 +267,40 @@ func (t *TerraformVariables) Load(f asset.FileFetcher) (found bool, err error) {
 
 	return true, nil
 }
+
+// injectInstallInfo adds information about the installer and its invoker as a
+// ConfigMap to the provided bootstrap Ignition config.
+func injectInstallInfo(bootstrap []byte) (string, error) {
+	config := &igntypes.Config{}
+	if err := json.Unmarshal(bootstrap, &config); err != nil {
+		return "", errors.Wrap(err, "failed to unmarshal bootstrap Ignition config")
+	}
+
+	var invoker string
+	if env, ok := os.LookupEnv("OPENSHIFT_INSTALL_INVOKER"); ok {
+		invoker = env
+	} else if user, err := user.Current(); err == nil {
+		invoker = user.Username
+	} else {
+		logrus.Warnf("Unable to determine username: %v", err)
+		invoker = "<unknown>"
+	}
+
+	config.Storage.Files = append(config.Storage.Files, ignition.FileFromString("/opt/openshift/manifests/kni-install.yml", "root", 0644, fmt.Sprintf(`---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kni-install
```

Comment:
openshift/cluster-version-operator#213 relies on this config map being named "openshift-install"

Reply:
This shouldn't cause any problems, and won't be part of the openshift/install PR